src/app/pages/system/CloudCredentials/cloudcredentials-form/cloudcredentials-form.component.ts

```diff
@@ -96,6 +109,10 @@ export class CloudCredentialsFormComponent {
       type: 'input',
       name: 'secret_access_key-S3',
       placeholder: T('Secret Access Key'),
+      tooltip: T('Amazon Web Services password. Create a new key\
+                  pair on the same AWS screen where the Access Key is\
```

Comment:
The second sentence is worded as an imperative, but really is (and should be) an if/then: "If the Secret Access Key cannot be found or remembered, go to My Account..."